routes/api.php

```diff
@@ -23,3 +24,13 @@
         Route::post('/', 'store')->name('store');
         Route::delete('/{city}', 'destroy')->name('destroy');
     });
+
+Route::controller(AirlineController::class)
+    ->prefix('airlines')
+    ->name('airlines.')
+    ->group(function () {
```

Comment:
You can group the `put` and `delete` routes like so (for better readability, place it below the other routes):
```
Route::prefix('{airline}')->group(static function () {
    Route::put('/', 'update')->name('update');
    Route::delete('/', 'destroy')->name('destroy');
});
```
The `static` keyword before the `function` keyword is for using less memory.